src/native/libs/System.Globalization.Native/pal_placeholders.c

```diff
@@ -0,0 +1,276 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#include <stdio.h>
+#include <assert.h>
+
+#include "pal_icushim_internal.h"
+#include "pal_icushim.h"
+#include "pal_calendarData.h"
+#include "pal_casing.h"
+#include "pal_collation.h"
+#include "pal_locale.h"
+#include "pal_localeNumberData.h"
+#include "pal_localeStringData.h"
+#include "pal_normalization.h"
+#include "pal_timeZoneInfo.h"
+
+#ifdef DEBUG
+#define assert_err(cond, msg, err) do \
```

Comment:
We `#include "pal_utilities.h"` everywhere else to get these assert definitions. Why can't we do it here?

Reply:
The ios runtime build builds globalization into its runtime as opposed to using libSystem.Globalization. I couldn't figure out how to generate pal_config.h, which is required when using pal_utilities.

Reply:
Add it to the cleanup list (https://github.com/dotnet/runtime/issues/96328) ?